src/components/AlertBanner/AlertBanner.tsx

```diff
@@ -71,7 +72,10 @@ const AlertBanner: VibeComponent<AlertBannerProps> & {
     }, [originalChildren, isDarkBackground]);
 
     return (
-      <div
+      <Text
+        element="div"
```

Comment:
is this still relevant after you change to make `Text` render `div` by default?

Reply:
removed